src/functional/program_processors/codegens/gtfn/gtfn_module.py

```diff
@@ -48,25 +52,87 @@ def __call__(
     ) -> stages.ProgramSource[languages.Cpp, languages.LanguageWithHeaderFilesSettings]:
         """Generate GTFN C++ code from the ITIR definition."""
         program = inp.program
-        parameters = tuple(
+        parameters: list[
+            interface.ScalarParameter | interface.BufferParameter | interface.ConnectivityParameter
+        ] = [
             get_param_description(program_param.id, obj)
             for obj, program_param in zip(inp.args, program.params)
+        ]
+        for name, connectivity in inp.kwargs["offset_provider"].items():
+            if isinstance(connectivity, Connectivity):
+                parameters.append(
+                    interface.ConnectivityParameter(
+                        "__conn_" + name.lower(), connectivity.origin_axis.value, name
```

Comment:
```suggestion
                        "conn_" + name.lower(), connectivity.origin_axis.value, name
```
`__something` is reserved in c++